### ISSUE TITLE
Saving space by optimizing CSS resets?

### ISSUE THREAD

**Issue by aschrijver** [KyleAMathews/typography.js#184]
I really like typography.js but I notice there are a lot of resets that take up space in the output, which - when I remove them - do not alter the page layout (at least on FF).

I am not a CSS expert, but I am wondering whether the resets can be optimized to take up less space.
Looks like they are set in [createStyles.js](https://github.com/KyleAMathews/typography.js/blob/master/packages/typography/src/utils/createStyles.js) where there are individual clauses for `margin-top:0`, `padding-left:0`, etc.

When testing with Ocean Beach the size of the output in production goes from 9164 bytes to 6660 bytes when removing the individual resets, a 27% reduction.

**Comment by TylerBarnes** [comment]
Is your server gzipping the file?

**Comment by aschrijver** [comment]
Yes, I am using Gatsby. The number of bytes mentioned are reflecting the uncompressed sizes.

**Comment by TylerBarnes** [comment]
Probably when gzipped the difference will be minimal or nonexistant. I think I remember reading that that's why Kyle wont optimize it. I think it's easier for typographyjs to overwrite these individual declarations. If you want to remove the normalize.css you can add the option `includeNormalize: false` to your config.

**Comment by KyleAMathews** [comment]
Happy to take a straightforward PR that fixes this but as Tyler says, the difference is negligible after gzip.

**Comment by aschrijver** [comment]
Thank you @TylerBarnes @KyleAMathews I'll try `includeNormalize`. I'll leave this open for PR'ing (but you can certainly close, if you want)

**Comment by gmossong** [comment]
On a similar note, Typography adds a lot of selectors that many sites will never need. Is there a proper way to include only the selectors you want or exclude the ones you don't? For example, a simple portfolio site may only use <h1> & <h2> selectors so there'd be no need for <h3>, <h4>, <h5> or <h6> selectors.
